vpc.tf

```diff
@@ -19,3 +23,10 @@ module "vpc" {
     "Name" = "${var.name}-${var.environment}"
   })
 }
+
+resource "aws_flow_log" "aws_flowlogs" {
+  log_destination      = "${data.aws_s3_bucket.cimpress-security-flowlogs.arn}/${module.vpc.vpc_id}/"
```

Comment:
For consistency with everything else, would you mind removing the extra spaces used to line things up?  (when I was modifying things I eventually got tired of having to re-line everything up, so just removed all the alignment)

Reply:
Re the whitespace, whiteout it it would fail `terraform fmt`, which we use to run a lint check on code before the validate/plan stage.

Note: When importing a module, terraform fmt does not lint check the remote code, only the local, so this won't impact us in GitLab. 

```
> terraform fmt -diff -check
vpc.tf
--- old/vpc.tf
+++ new/vpc.tf
@@ -9,15 +9,15 @@
 module "vpc" {
   source = "terraform-aws-modules/vpc/aws"

-  name = "${var.name}-${var.environment}"
-  cidr = var.cidr_block
-  azs = data.aws_availability_zones.available.names
-  private_subnets = var.private_subnets
-  public_subnets = var.public_subnets
-  enable_dns_support = true
+  name                 = "${var.name}-${var.environment}"
+  cidr                 = var.cidr_block
+  azs                  = data.aws_availability_zones.available.names
+  private_subnets      = var.private_subnets
+  public_subnets       = var.public_subnets
+  enable_dns_support   = true
   enable_dns_hostnames = true
-  enable_nat_gateway = true
-  single_nat_gateway = true
+  enable_nat_gateway   = true
+  single_nat_gateway   = true

   tags = merge(local.all_tags, {
     "Name" = "${var.name}-${var.environment}"
```